FRANKDOC.md

```diff
@@ -0,0 +1,30 @@
+# Frank!Doc user manual for contributors
+
+The Frank!Doc provides reference information for Frank developers as explained in [CONTRIBUTING.md](./CONTRIBUTING.md). It is produced by a doclet that assembles the Frank!Doc from the Java sources. You can control the Frank!Doc using custom JavaDoc tags and using Frank!Doc-related Java annotations. These annotations and tags are summarized in the table below:
+
+
+| Kind | Appears on | Name | Function |
+| ---- | ---------- | ---- | -------- |
+| Java annotation | Class or interface | @FrankDocGroup | Define group as shown in top-left of webapp. |
```

Comment:
formatting voor @FrankDocGroup ontbreekt

Reply:
Ik noem nu gelijk de velden `name` en `order`. Is dat wat je wilt?

Reply:
Ik bedoelde dat de back-quotes rondom @FrankDocGoup ontbreken

Reply:
Gedaan.